gym_sts/envs/single_combat.py

```diff
@@ -0,0 +1,54 @@
+from gym_sts.spaces.observations import Observation
+
+from .base import SlayTheSpireGymEnv
+
+
+class SingleCombatSTSEnv(SlayTheSpireGymEnv):
+    def __init__(self, *args, enemy: str = "3_Sentries", **kwargs):
+        if "value_fn" not in kwargs:
+            kwargs["value_fn"] = single_combat_value
+        super().__init__(*args, **kwargs)
+        self.enemy = enemy
+
+    def reset(self, *args, **kwargs):
+        res = super().reset(*args, **kwargs)
+
+        obs = self.communicator.basemod(f"fight {self.enemy}")
+        assert obs.in_combat
+        self.observation_cache.append(obs)
+
+        if type(res) == tuple and len(res) == 2:
```

Comment:
We can include the `return_info` kwarg in this method's signature, and check its value here instead.

Reply:
Done